ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v1.admin;
+
+option java_outer_classname = "UserManagementServiceOuterClass";
+option java_package = "com.daml.ledger.api.v1.admin";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1.Admin";
+
+
+
+// --> See USER_MANAGEMENT_DESIGN.md for status and notes <-- //
+
+
+
+// Manage users and their rights for interacting with the Ledger API served by participant node.
+//
+// The API roughly follows the Google API style guidelines for resource-oriented design
+// (https://cloud.google.com/apis/design/resources) with the following simplifications to 
+// ease its implementation:
+// 
+// 1. List methods do not (yet) support pagination, 
+//    as we expect to have fewer than 10k users and 1k rights per user.
+// 
+// 2. Resources are not named as per the Google API style guide, 
+//    as this would be in contrast with the other Ledger API services.
+//
+service UserManagementService {
+
+    // Create a new user, failing if it already exists.
+    rpc CreateUser (CreateUserRequest) returns (User);
```

Comment:
Does this also allocate a party?

Reply:
No it does not.

Reply:
It still does not, but the service now tracks a primary_party per user.

Reply:
So this means the expected use is that I first allocate and then create a user? Is there a reason why we don’t allow to create the primary party as part of the createuser request? Having looked at the Daml Script API, having to create the party first seems a bit odd.